lib/autoupdate/agent/config.go

```diff
@@ -203,3 +289,36 @@ func (i InstallFlags) String() string {
 	}
 	return "Unknown"
 }
+
+// DirFlag returns the directory path representation of a single InstallFlag flag, or "unknown".
+func (i InstallFlags) DirFlag() string {
+	switch i {
+	case 0:
+		return ""
+	case FlagEnterprise:
+		return "ent"
+	case FlagFIPS:
```

Comment:
Are these bit flags? Any risk of us converting to "unknown" if both `FlagEnterprise` and `FlagFIPS` is set? Maybe this helps?
```suggestion
	case FlagFIPS, FlagEnterprise|FlagFIPS:
```

Reply:
These are bit flags. This method is just intended for single-bit contexts, similar to `String()`.